### DISCUSSION TITLE
Are there any other ways to donate?

### DISCUSSION THREAD

**Discussion by flexagoon** [deckerst/aves#121] [General]
Just anything except paypal, since paypal is hard to use in Russia

**Comment by deckerst** [comment]
Nothing is set up, but I'm open to suggestions. Do you know anything convenient for the donor and the recipient?

**Reply by flexagoon** [reply]
The easiest way is probably GitHub Sponsors. And I've heard that Librepay is also pretty easy to set up.

**Reply by deckerst** [reply]
I think GitHub Sponsors is not available in my country of residence, but I'll take a look at these options, thanks.

**Reply by flexagoon** [reply]
Other options that I've seen are OpenCollective and StreamLabs, but I don't know how easy they are

**Reply by deckerst** [reply]
I set up an account at Liberapay, with a link [here](https://github.com/deckerst/aves/blob/develop/README.md#donations). But it seems Liberapay is only for recurring donations. Is it really what you want?

**Reply by flexagoon** [reply]
Liberapay FAQ says this:

> One-time donations aren't properly supported yet, but you can discontinue your donation immediately after the first payment.

**Reply by deckerst** [reply]
I had to close the Stripe account that was backing Liberapay (Liberapay backed by Paypal is still there, but I suppose it's not much use to you). My country of residence is badly supported by international financial services :/

GitHub Sponsors can distinguish between country of residence and country of bank account. They don't fully support my country of residence, but I do have a bank account in a country that they fully support, so I've added myself on the Sponsors waitlist. We'll see.

**Reply by flexagoon** [reply]
@deckerst wait, I still am subscribed to you on Liberapay. Does this mean that I have to cancel ny subscription?

**Reply by deckerst** [reply]
@flexagoon you're free to stop anytime of course, but so far your contributions didn't get lost and they reached me. Thanks again for your support.

It seems Liberapay/Stripe bundle and transfer in advance contributions. On 2021/11/17 I received the equivalent of 12 weeks of your contributions. So far it's been 9 weeks since the beginning of your subscription, so I'm curious what will happen in 4 weeks:
- maybe Stripe will continue to honor the subscription as it was set up, even though I closed my account since then,
- maybe Liberapay will let you know that contributing via Stripe is no longer possible and automatically stop your subscription,
- maybe Liberapay will automatically switch from Stripe to Paypal to make your contributions reach me, and it will be transparent to you.

No idea!

**Reply by kha84** [reply]
Tried to use LP but it looks like because of all known world events, cards issued by my country are not accepted abroad any longer. I'd wish to support this project with regular donations anyhow.  Probably if you have some time, you could investigate any cryptocurrencies options? I never used any of that myself, but they are known for being border-free. I'm guessing a lot of folks from some big northern country will follow that and appreciate your work with some material donations

**Reply by deckerst** [reply]
Don't worry, financial contributions are appreciated, but it's not the only way to contribute. You already helped with your ideas and bug reports, and I am grateful for that.

If you want to help further, maybe you could help with the Russian translation? The person who was maintaining it so far seems busy, and I figured you know both Russian and English... If you're interested, feel free to [email me](mailto:gallery.aves@gmail.com).

As for cryptos, I am not fond of them, sorry.

**Reply by flexagoon** [reply]
@deckerst have you considered setting up something like Crowdin or Weblate? It's not that hard for mobile apps afaik. I'm also from Russia, so I'm no longer able to donate on LP, but I'd love to help with translation.

**Reply by deckerst** [reply]
Regarding Crowdin/Weblate, it's probably possible to set it up for free, but for now I'm happy with how Aves translations are managed, that is, manually. If people want to contribute translations, they can and there's even a [guide](https://github.com/deckerst/aves/wiki/Contributing-to-Translations).

Regarding donations, I forgot to mention it but i recently moved to another country so:
- I changed the PayPal account,
- I created a Stripe account,
- Liberapay contributions can use either PayPal or Stripe,
- I set up GitHub Support.

**Reply by Nekothegamer** [reply]
> Nothing is set up, but I'm open to suggestions. Do you know anything convenient for the donor and the recipient?

I'd suggest you to use buy me a coffee, it can do both one time donations and recurring payments like Patreon (or liberapay like previously mentioned in this thread). 

I'd highly suggest it to you for the project, it's a really good platform for creator funding 

link:
https://www.buymeacoffee.com/

**Reply by deckerst** [reply]
Thanks for the suggestion, but now that Liberapay and GitHub Support are set up, isn't it redundant? Or is there some specific benefit to using buymeacoffee?

**Reply by Nekothegamer** [reply]
oh damn, you eventually got accepted to put the sponsor button on the repo, i saw that only now.
But nonetheless, buymeacoffee could still be useful in a way since it can do both subscriptions or single donations, so it could be useful for having a unified platform for supporting the project instead of two other different platforms that does thing separately, it's more convenient.

i don't know man you do you mine was only a suggestion.
